appintro/src/main/java/com/github/paolorotolo/appintro/internal/CustomFontCache.kt

```diff
@@ -12,21 +13,20 @@ internal object CustomFontCache {
     private val TAG = LogHelper.makeLogTag(CustomFontCache::class)
     private val cache = hashMapOf<String, Typeface>()
 
-    operator fun get(path: String?, ctx: Context): Typeface? {
+    operator fun get(ctx: Context, path: String?, fontCallback: ResourcesCompat.FontCallback) {
```

Comment:
```suggestion
    fun getFont(ctx: Context, path: String?, fontCallback: ResourcesCompat.FontCallback)
```

Please rename to `.getFont` and also remove the `operator` keyword.
Having a `get` operator will make the `[]` syntax really confusing. Also having `.getFont` will make the naming consistent with `ResourceCompat`.